.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+### Plugin Name
+[This name of the plugin you are having an issue with]
+
+### Plugin version
+[Version of Plugin in use]
```

Comment:
Plugins have versions?  How does one find the version?  That should be described here